ddtrace/contrib/pytest/_plugin_v2.py

```diff
@@ -525,6 +525,10 @@ def _pytest_terminal_summary_post_yield(terminalreporter, failed_reports_initial
 @pytest.hookimpl(hookwrapper=True, tryfirst=True)
 def pytest_terminal_summary(terminalreporter, exitstatus, config):
     """Report flaky or failed tests"""
+    from ddtrace.appsec._iast._pytest_plugin import print_iast_report
+
+    print_iast_report(terminalreporter)
```

Comment:
It's probably good to wrap this call in a try/except like the ones below, to avoid breaking the user pipeline if something goes wrong.